src/scenes/MerchStorePage/MerchStorePage.tsx

```diff
@@ -5,6 +5,8 @@ import { TitleM } from 'components/core/Text/Text';
 import ItemPreview from './ItemPreview';
 import Countdown from './Countdown';
 import { GLOBAL_FOOTER_HEIGHT } from 'contexts/globalLayout/GlobalFooter/GlobalFooter';
+// import OpenBracket from 'public/icons/open_bracket.svg';
```

Comment:
toss?

Reply:
lol this was to update the brackets to be in line with the gallery logo per Jess' comments but not gonna lie i couldn't reduce the size of them to save my life